app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,20 @@
   border-bottom: 1px solid gray;
 }
 
+.card {
+  height: 150px;
```

Comment:
Properties should be ordered background-color, border, color, cursor, display, font-size, height, transition, width